test/jdk/jdk/incubator/vector/templates/Unit-header.template

```diff
@@ -1500,6 +1500,17 @@ public class $vectorteststype$ extends AbstractVectorTest {
 #end[intOrLong]
     }
 
+    static $type$ REVERSE_BYTES_scalar($type$ a) {
+#if[intOrLong]
+        return $Wideboxtype$.reverseBytes(a);
+#else[intOrLong]
+#if[short]
+        return (short) ((a << 8) | (a >>> 8));
+#else[short]
+        return (byte) a;
+#end[short]
+#end[intOrLong]
+    }
```

Comment:
Is it possible to move these scalar operations into the `gen-template.sh `? It can also generate different codes for different types. And the codes can be generated finally by `Unit-Unary-Scalar-op.template`. I guess it doesn't need to add the same codes both in `Perf-Scalar-header.template` after then.